common.mk

```diff
@@ -35,6 +35,10 @@ SUBSTRATE_PLUGIN_OS=${PROJECT_REL_DIR}/build/substratehcp-$(1)-amd64-${SUBSTRATE
 SUBSTRATE_PLUGIN_LINUX=$(call SUBSTRATE_PLUGIN_OS,linux)
 SUBSTRATE_PLUGIN_DARWIN=$(call SUBSTRATE_PLUGIN_OS,darwin)
 SUBSTRATE_PLUGIN=${SUBSTRATE_PLUGIN_DARWIN} ${SUBSTRATE_PLUGIN_LINUX}
+# Cross-platform debugging. SUBSTRATE_PLUGIN usage is currently broken on linux
```

Comment:
I think the `SUBSTRATE_PLUGIN` variable name is probably a bit misleading, but I believe it should just be used just for downloading the two plugin versions.  I think the bug you are seeing isn't directly related to this, though forcing the values could help.  Do you mind opening an issue for this one instead?